public/apps/configuration/app-router.tsx

```diff
@@ -88,22 +84,27 @@ const ROUTE_MAP: { [key: string]: RouteItem } = {
   },
 };
 
-const ROUTE_LIST = [
-  ROUTE_MAP.getStarted,
-  ROUTE_MAP[ResourceType.auth],
-  ROUTE_MAP[ResourceType.roles],
-  ROUTE_MAP[ResourceType.users],
-  ROUTE_MAP[ResourceType.serviceAccounts],
-  ROUTE_MAP[ResourceType.permissions],
-  ROUTE_MAP[ResourceType.tenants],
-  ROUTE_MAP[ResourceType.auditLogging],
-  ROUTE_MAP[ResourceType.tenantsConfigureTab],
-];
+const getRouteList = (multitenancyEnabled: boolean) => {
+  return [
+    ROUTE_MAP.getStarted,
+    ROUTE_MAP[ResourceType.auth],
+    ROUTE_MAP[ResourceType.roles],
+    ROUTE_MAP[ResourceType.users],
+    ROUTE_MAP[ResourceType.serviceAccounts],
+    ROUTE_MAP[ResourceType.permissions],
+    ...(multitenancyEnabled ? [ROUTE_MAP[ResourceType.tenants]] : []),
+    ROUTE_MAP[ResourceType.auditLogging],
+    ...(multitenancyEnabled ? [ROUTE_MAP[ResourceType.tenantsConfigureTab]] : []),
```

Comment:
we should remove this "ghost tab"

Reply:
Done in: [9404930](https://github.com/opensearch-project/security-dashboards-plugin/pull/1960/commits/940493040dc112c8a40f0e551fe58b467f7826d3)